twake/frontend/src/app/services/Observable/Observable.ts

```diff
@@ -49,41 +52,52 @@ export default class Observable {
       const changes = await this.getChanges(watcher);
       if (changes.didChange) {
         //If things changed
-        watcher.callback(changes.changes);
+        watcher.callback(
+          //Could be imporved ?
+          typeof changes.changes === 'object' && changes.changes?.constructor?.name != 'Array'
```

Comment:
Maybe ? 
`typeof changes.changes === 'object' && !changes.changes instanceof Array`

(I'm not sure about this)